src/configure/helper/redirectLinkHelper.ts

```diff
@@ -0,0 +1,38 @@
+import { RestClient } from "typed-rest-client";
+
+export enum ServiceFramework {
+    Moda,
+    Vssf
+}
+
+export class RedirectLinkHelper {
```

Comment:
this should be called `RemoteServiceUrlHelper` so as not to tie it with underlying logic